Switch to BSD Source Code Attribution license

PandaScience opened the pull request:
Switch from BSD 3-Clause to BSD Source Code Attribution to prevent confusion around inclusion of copyright notices when distributing in binary form, i.e. compiled PDFs.

PandaScience committed:
Switch to BSD Source Code Attribution license

PandaScience commented:
With regards to the recently started license discussion in #49 I'd like to ask all previous contributors to clearly state their consent or refusal to this change in license as a comment below.

The licenses in questions are [from] **BSD 3-Clause** [to] **BSD Source Code Attribution**, i.e. a migration to a more permissive license (see #49 for details).

Rejecting this change would regrettably force me to remove individual (genuine) contribution(s) for upcoming versions due to possible license incompatibility, except when they are trivial additions according to common sense, like fixing a typo, duplicating existing code just to add new icons etc. 

:pray:  @ifokkema @Bertbk @mikelyou @dverbeir @bigbenbeer @esben @DoctorD90 @Hespe  :pray:

Hespe commented:
No problems with the change from my side.

esben commented:
You have my consent to the the proposed change of the license.

ifokkema commented:
Of course, you have my consent, too! :hugs:

bigbenbeer commented:
No problem with the changes from me.

dverbeir commented:
You have my consent too to migrate to BSD Source Code Attribution license.

DoctorD90 commented:
Objection, Your Honor! - Sorry, I've always wanted to say that. It's okay for me too!

PandaScience commented:
Friendly reminder @Bertbk and @mikelyou :pray:

Bertbk commented:
Ok for me! Sorry for the delay :-)

PandaScience committed:
Pre-license change cleanup

Remove genuine contributions from authors with missing consent in #51
due to possible license incompatibility.
